transport/http/peer.go

```diff
@@ -167,6 +187,13 @@ func (p *httpPeer) MaintainConn() {
 }
 
 func (p *httpPeer) setStatus(status peer.ConnectionStatus) {
+	p.transport.logger.Info(
+		"connection status change",
+		zap.String("status", status.String()),
+		zap.String("address", p.Peer.Identifier()),
```

Comment:
Can we use `peer` instead of `address` (same with the other transport peers)? I think this would make the log clearer to users.